prisma/migrations/20240112121757_init/migration.sql

```diff
@@ -0,0 +1,17 @@
+-- CreateTable
```

Comment:
I was having trouble running prisma migrations, and got an error to the effect of "the `users` table does not exist". 

After removing the existing `add_timestamps` migration, and re-running `prisma migrate`, this migration was output, which contains the base schema... was this meant to be here all along?

Either way, we might have trouble deploying this to `baas.stripe.dev` as-is. We should decide if we want to keep this migration (and manually massage the current production deployment's schema migration history so that it doesn't fail on this new one), or fix some other way